schemas/metadata/error/error.1.schema.json

```diff
@@ -0,0 +1,108 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "http://jsonschema.net",
+  "properties": {
+    "args": {
+      "type": "string"
+    },
+    "content_length": {
+      "type": "string"
+    },
+    "date": {
+      "type": "string"
+    },
+    "dnt": {
+      "type": "string"
+    },
+    "error_message": {
+      "type": "string"
+    },
+    "error_type": {
+      "type": "string"
+    },
+    "exception_class": {
+      "type": "string"
+    },
+    "geo_city": {
+      "type": "string"
+    },
+    "geo_country": {
+      "type": "string"
+    },
+    "geo_subdivision1": {
+      "type": "string"
+    },
+    "geo_subdivision2": {
+      "type": "string"
+    },
+    "host": {
+      "type": "string"
+    },
+    "method": {
+      "type": "string"
+    },
+    "payload": {
+      "format": "bytes",
```

Comment:
@acmiyaguchi i'm unclear on whether the transpiler already does the right thing, or we need to add supports for the bytes format somehow